src/Sarif.Viewer.VisualStudio.UnitTests/ErrorList/SarifErrorListItemTests.cs

```diff
@@ -388,7 +387,27 @@ public void SarifErrorListItem_ResultMessageFormat_NoTrailingPeriod()
 
             SarifErrorListItem item = MakeErrorListItem(result);
             item.Message.Should().Be(s1);
-            item.ShortMessage.Should().Be(s1);
+            item.ShortMessage.Should().Be($"{s1}.");
```

Comment:
>item.ShortMessage.Should().Be($"{s1}."); [](http://example.com/codeflow?start=12&length=40)

is it by design we want to add a "."

Reply:
I see we only adding "." to ShortMessage but not Message
Looks like not consistent.
ShortMessage should not be longer than Message

Reply:
You are right this is the behavior of existing Sarif-Sdk extension methods. We need our own methods to handle the differences.

Reply:
With a new implementation this is fixed